tests/TestCase.php

```diff
@@ -74,10 +74,10 @@ protected function defaults(): array
         return $defaults;
     }
 
-    protected function assertLocalHas(array $settings): void
+    protected function assertLocalHas(array $settings, string $settings_path = 'scout-users.php'): void
```

Comment:
`$settingsPath` instead maybe?

Reply:
Yeah, here it should be `$settingsPath`.